linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny_root/rule.yml

```diff
@@ -103,3 +103,6 @@ warnings:
         be shown in the remediation report.
         If the system supports the <tt>/etc/security/faillock.conf</tt> file, the pam_faillock
         parameters should be defined in <tt>faillock.conf</tt> file.
+
+srg_requirement: |-
+    {{{ full_name }}} must automatically lock the root account until the root account is released by an administrator when three unsuccessful logon attempts occur during a 15-minute time period.
```

Comment:
No newline at the end of file

Reply:
I believe it github explicitly mentions when there is no newline at the end o the file, which is not the case here.

Reply:
I went commit by commit and the red sign is shown there, but when I see the whole diff it isn't shown there, probably it was fixed by a later commit, so if there is a newline at the end of file then please disregard it.